web/src/composebox_typeahead.ts

```diff
@@ -900,13 +887,21 @@ export function get_candidates(
         return typeahead_helper.sort_slash_commands(matches_list, token);
     }
 
-    if (ALLOWED_MARKDOWN_FEATURES.stream && current_token.startsWith("#")) {
+    // Not a stream if it ends with '> some text'
+    if (
+        ALLOWED_MARKDOWN_FEATURES.stream &&
+        current_token.startsWith("#") &&
+        !/>[a-zA-Z0-9\s]*$/.test(current_token)
```

Comment:
Do you need this? What breaks if you don't include this regex here?

Reply:
Oh yeah, I see. I think a better way to handle this would be to move the stream check after the stream>topic check, then you can avoid adding the extra regex.